tests/tpch/conftest.py

```diff
@@ -205,12 +205,17 @@ def spark_setup(cluster, local):
 
     spark_dashboard: Url
     if local:
+        cluster.close()
         from pyspark.sql import SparkSession
 
         # Set app name to match that used in Coiled Spark
         spark = (
             SparkSession.builder.appName("SparkConnectServer")
+            .config("spark.driver.host", "127.0.0.1")
             .config("spark.driver.bindAddress", "127.0.0.1")
```

Comment:
Of course both configs have to be set. Because why not. This appears to be the reason for some jobs to timeout during collect (see https://github.com/coiled/benchmarks/pull/1492#issuecomment-2014849506)